qiita_core/support_files/server.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
I think this is a bad idea if we are targeting this for single user laptop use as well, as I can foresee a lot of people freaking out when they first start the system and see that error.